biogeochem/EDPhysiologyMod.F90

```diff
@@ -1079,7 +1117,18 @@ subroutine PlantGrowth( currentSite, currentCohort, bc_in )
     end if
     
     if (EDPftvarcon_inst%evergreen(ipft) == 1)then
-       currentCohort%leaf_md = currentCohort%bl / EDPftvarcon_inst%leaf_long(ipft)
+       if(use_leaf_age==itrue)then
+          if(currentSite%dstatus==1)then
```

Comment:
I think up to this point, we have been using dstatus in the phenology routine to flag if leaves are on or off, with instantaneous effects.   It seems that this is a way to make leaves drop over a period of time when drought status is triggered?